foundation_cms/legacy_apps/wagtailpages/wagtail_hooks.py

```diff
@@ -132,26 +128,6 @@ def manage_index_pages_cache(request, page):
         parent.clear_index_page_cache(locale)
 
 
-@hooks.register("before_delete_page")
-def delete_historical_ab_tests_before_page_deletion(request, page):
-    """
-    Fixes a bug where historical A/B tests prevent page deletion by removing them beforehand.
-    Active A/B tests will still block deletion. For more info, see:
-    https://github.com/wagtail-nest/wagtail-ab-testing/issues/90
-    """
-    # First, check if there is an active A/B test; if so, block deletion with an error message.
-    if AbTest.objects.filter(page=page, status=AbTest.STATUS_RUNNING).exists():
-        # Redirect to the parent page and display an error message in the CMS
-        messages.error(request, "This page has an active A/B test and cannot be deleted.")
-        return redirect("wagtailadmin_explore", page.get_parent().id)
-
-    # If there are no active A/B tests, delete all historical A/B tests for the page
-    # to prevent them from blocking the page deletion.
-    historical_ab_tests = AbTest.objects.filter(page=page).exclude(status=AbTest.STATUS_RUNNING)
-    if historical_ab_tests.exists():
-        historical_ab_tests.delete()
-
-
```

Comment:
Wagtail v0.12.0 solves this bug! 🙌  [Link to PR](https://github.com/wagtail-nest/wagtail-ab-testing/issues/90)